parser.y

```diff
@@ -1146,6 +1152,10 @@ import (
 	EnforcedOrNotOrNotNullOpt	"{[ENFORCED|NOT ENFORCED|NOT NULL]}"
 	Match			"[MATCH FULL | MATCH PARTIAL | MATCH SIMPLE]"
 	MatchOpt		"optional MATCH clause"
+	MaxMinutesOpt	"MAX_MINUTES num(int)"
+	MaxIndexNumOpt	"MAX_IDXNUM clause"
+	PerTable 		"Max index number PER_TABLE"
+	PerDB			"Max index number PER_DB"
```

Comment:
Please format these lines.

Reply:
Oh sorry 
But what tools are helpful? I always seem to be puzzled by `tab` in `parser.y`😂

Reply:
We are planning to develop such a tool. See #670.

The `parser.go` is still in conflict, you can run `make parser` to resolve it.